src/index.ts

```
@@ -1,28 +1,35 @@
-import { Beautifier, Language, BeautifierBeautifyData } from "unibeautify";
+import { Beautifier, Language, BeautifierBeautifyData, DependencyType, NodeDependency } from "unibeautify";
 import * as readPkgUp from "read-pkg-up";
 import options from "./options";
-const CLIEngine = require("eslint").CLIEngine;
 const { pkg } = readPkgUp.sync({ cwd: __dirname });
 export const beautifier: Beautifier = {
   name: "ESLint",
   package: pkg,
+  dependencies: [
+    {
+      type: DependencyType.Node,
+      name: "ESLint",
+      package: "eslint",
+    }
+  ],
   options: {
     JavaScript: options.JavaScript,
   },
-  beautify(data: BeautifierBeautifyData) {
+  beautify({ text, options, dependencies }: BeautifierBeautifyData) {
     return new Promise<string>((resolve, reject) => {
+      const { CLIEngine } = dependencies.get<NodeDependency>("ESLint").package;
       const cli = new CLIEngine({
         fix: true,
         parserOptions: {
           ecmaVersion: 6,
         },
-        rules: data.options,
+        rules: options,
       });
-      const text = cli.executeOnText(data.text).results[0].output;
-      if (text) {
-        return resolve(text);
+      const result = cli.executeOnText(text).results[0].output;
+      if (result) {
+        return resolve(result);
       } else {
-        return resolve(data.text);
+        return resolve(text);
```

I wonder if we should throw an error here instead? As part of another PR.